app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -347,7 +347,11 @@ private function getAddressHints($quote, $checkoutType)
             $hints['virtual_terminal_mode'] = true;
         }
 
-        $hints['prefill'] = $prefill;
+        // Skip pre-fill for Apple Pay related data.
+        if (!$prefill['email'] == 'fake@email.com' && $prefill['phone'] == '1111111111') {
```

Comment:
I don't think this will work as intended. The negation should appear in the comparison operator, or we should group the comparisons and negate the set. I think this update should accomplish the goal:
`if (!($prefill['email'] == 'fake@email.com' && $prefill['phone'] == '1111111111'))
`

Reply:
Shoot good catch.